peps/pep-0745.rst

```diff
@@ -0,0 +1,72 @@
+PEP: 745
+Title: Python 3.14 Release Schedule
+Author: Hugo van Kemenade <hugo@python.org>
+Status: Active
+Type: Informational
+Topic: Release
+Created: 24-Apr-2024
+Python-Version: 3.14
+
+
+Abstract
+========
+
+This document describes the development and release schedule for
+Python 3.14.
+
+Release Manager and Crew
+========================
+
+- 3.14 Release Manager: Hugo van Kemenade
+- Windows installers: Steve Dower
+- Mac installers: Ned Deily
+- Documentation: Julien Palard
+
+
+Release Schedule
+================
+
+3.14.0 schedule
+---------------
+
+The dates below use a 17-month development period that results
+in a 12-month release cadence between feature versions, as defined by
+:pep:`602`.
+
+Expected:
+
+- 3.14 development begins: Tuesday, 2024-05-07
+- 3.14.0 alpha 1: Tuesday, 2024-10-15
+- 3.14.0 alpha 2: Tuesday, 2024-11-19
```

Comment:
FWIW, I do releases on Tuesdays because those days work out best for me: there are several Dutch holidays on Mondays, I occasionally travel on Mondays, and I usually don't have a chance to spend much time to prepare on Sundays. I didn't realize Tuesday was more convenient until well into the 3.12 release cycle, so I would recommend re-evaluating whatever day you choose after the first few releases.

(Other than the two release dates that are mandated by pop culture of course.)

Reply:
Thanks, I'll see how it goes and adjust as needed.

I think a Monday can be useful for last minute things to be merged. Tuesdays are usually free for me, the Docs Community monthly meetings are the first Tuesday of the month, which would only clash with beta 1, but let's see. I noticed a fair bit of flexibility in the 3.12/3.13 schedules :)